pkg/datapath/loader/template.go

```diff
@@ -216,10 +216,8 @@ func elfVariableSubstitutions(ep datapath.Endpoint) map[string]uint32 {
 
 	if ipv6 := ep.IPv6Address().AsSlice(); ipv6 != nil {
 		// Corresponds to DEFINE_IPV6() in bpf/lib/utils.h
-		result["LXC_IP_1"] = sliceToBe32(ipv6[0:4])
-		result["LXC_IP_2"] = sliceToBe32(ipv6[4:8])
-		result["LXC_IP_3"] = sliceToBe32(ipv6[8:12])
-		result["LXC_IP_4"] = sliceToBe32(ipv6[12:16])
+		result["LXC_IP_1"] = sliceToBe32(ipv6[0:8])
```

Comment:
Yep, the way it's implemented looks a bit strange to me. Instead of assigning individual bytes, why don't we write a single value to `v6addr->p1`, which is a u32?  @borkmann